packages/orchestration/src/orchestration-types.ts

```diff
@@ -63,6 +63,17 @@ export interface OrchestrationModuleConfig {
   llm: LlmModuleConfig;
   /**
    * Filtering module configuration.
+   * Construct filter configuration for both input and output filtering using convenience functions.
```

Comment:
```suggestion
   * Construct filter configuration for both input and output filters using convenience functions.
```

[pp] I would mention convenience functions as a separate line.